server/speech.js

```diff
@@ -0,0 +1,106 @@
+// This endpoint handles server calls to get transcript of an audio file
+// We check the database for an existing transcript, and if we find none
+// we use the IBM Watson api to transcribe one from the wav file on s3 and 
+// store it
+function transcribeEndpoint(pool, s3, watsonConfig, request, response) {
+  const {audioID} = request.params;
+  
+  const values = [audioID];
+  const sql = `
+    SELECT transcript
+    FROM transcripts
+    WHERE 1=1
+      AND audio_id = $1`;
+
+  pool.query(sql,values)
+    .then(results => {
+      if ((results.rowCount >= 1) && (results.rows[0].transcript !== "")){
```

Comment:
Would adding `LIMIT 1` to the SQL simplify these checks?

Reply:
So I think that since I use the 0th item in rows anyways, it is probably fine to have `LIMIT 1`. 

Do you think it would be better to use `LIMIT 1` and just have the if statement be 
```
if ((results.rowCount === 1) && (results.rows[0].transcript !== "")){
```

I'm not sure if I'm missing the point.